src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -291,7 +291,7 @@ $(document).ready(function() {
         toggleCollapse(this, panels);
     });
 
-    $('a[id^="collapse-panels-button-team-"]').on('click', function() {
+    $('.panel.panel-success').on('click', 'a[id^="collapse-panels-button-team-"]', function() {
```

Comment:
No, it shouldn't, because those buttons are loaded with the page, while "collapse-panels-button-team-*" are loaded asynchronously